src/core/appinterface.cpp

```diff
@@ -43,6 +44,21 @@ AppInterface::AppInterface( QgisMobileapp *app )
 {
 }
 
+QObject *AppInterface::createHttpRequest( bool autoDelete ) const
+{
+  auto *req = new QFieldXmlHttpRequest();
```

Comment:
-1 on use of auto here other than very limited usages, this one is not.

An interesting project for the first half of 2026: a c++ guideline for our growing nb of core developers :)